src/components/Person/Person.jsx

```diff
@@ -1 +1,21 @@
-// export const Person = ({ person }) => ();
+export const Person = ({ person }) => {
+  const { name, age, sex, isMarried, partnerName } = person;
+  const spouse = sex === 'f' ? 'husband' : 'wife';
+  let spouseStatus = <p className="Person__partner">I am not married</p>;
+
+  if (isMarried) {
+    spouseStatus = (
+      <p className="Person__partner">
+        {partnerName} is my {spouse}
+      </p>
+    );
+  }
```

Comment:
- It's better to use const and you can do a check for `isMarried` directly in return statement instead of assign to the variable
- `<p className="Person__partner">` in both cases, only the content is different